Explorer/Assets/DCL/Character/CharacterMotion/Animation/AvatarAnimationEventsHandler.cs

```diff
@@ -96,6 +101,8 @@ private void PlayStepSoundForFoot(Transform footTransform)
                 case MovementKind.Jog:
                     if (currentTime - lastFootstepTime > jobIntervalSeconds)
```

Comment:
I know it's not related to this PR, but can we fix the typo in `jobIntervalSeconds` to `jogIntervalSeconds`? 😅